docs/overview.md

```diff
@@ -1,44 +1,38 @@
-# Atlas overview
+# Atlas technical overview
 
-Atlas is a content consumption and publication app for the Joystream platform. This document outlines conventions/tools/services used to make Atlas work.
+This document tries to outline all the conventions/tools/services used to make Atlas work.
 
 ## Conventions
 
 ### Some of the used tools/tech
 
 - React
 - Typescript
-- GraphQL (+ [Relay-style pagination](https://graphql.org/learn/pagination/))
-- ESLint, Prettier - for enforcing common code style
+- GraphQL (+ [Relay-style pagination](https://graphql.org/learn/pagination/)) with [Apollo Client](https://www.apollographql.com/docs/react/)
```

Comment:
Maybe we could also mention Zustand here?

Reply:
Mentioned